mitodl/open-discussions
# remove podcast frontpage feature flag

#### Pre-Flight checklist



#### What are the relevant tickets?

closes #3189 

#### What's this PR do?

this removes the `PODCAST_FRONTPAGE` feature flag, since we've turned it on in production and kept it on for several months (and have no plans to turn it off).

I removed the backend definition of the flag and then the places where it was used to control rendering on the frontend as well.

#### How should this be manually tested?

You should be able to start your dev environment and navigate to `/podcasts` without any issues. Everything should work as normal on that page.
